### PR TITLE
LL-5136 (Account/Potfolio): cta actions reworked

### PR DESCRIPTION
**Pull request by LFBarreto**
(Account/Potfolio): cta actions reworked
Added buy / swap actions on portfolio + reworked actions on account page with some sticky animation on scroll.

https://user-images.githubusercontent.com/11752937/116090017-b87cd180-a6a3-11eb-9077-e4e99fbe3dc6.mp4

### Type

Feature
### Context

[LL-5136]
### Parts of the app affected / Test plan

Portfolio page
Account page -> look for:
- empty account 
- tezos account
- algorand account or account that cant buy or swap
- account that can buy or swap or wallet connect or lend

[LL-5136]: https://ledgerhq.atlassian.net/browse/LL-5136

### CODE REVIEW

**Review by juan-cortes — DISMISSED** [review]

File: src/components/FabAccountButtonBar.js [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+            onClose={() => setDisplayedActions()}
+            onModalHide={() => goToNext()}
+            containerStyle={styles.modal}
+          >
+            {displayedActions === "lending" && account && (
+              <LendingBanners account={account} />
+            )}
+            {!!displayedActions &&
+              actions[displayedActions] &&
+              actions[displayedActions].map((a, i) =>
```
Comment (JunichiSugiura) [inline src/components/FabAccountButtonBar.js:166]: could be as simple as `actions[displayedActions]?.map(...)` with optional chaining i think.

File: src/components/FabAccountButtonBar.js [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+function FabAccountButtonBar({
+  buttons,
+  actions,
+  account,
+  parentAccount,
+}: Props) {
+  const { colors } = useTheme();
+  const navigation = useNavigation();
+  const [next, setNext] = useState();
```
[the remark this answers is not in the document]
Comment (LFBarreto) [reply src/components/FabAccountButtonBar.js:56]: this state logic was just ported from what we had previously. not sure a reducer would make it easier to understand :thinking:

File: src/screens/Portfolio/index.js [hunk 3]
```diff
@@ -194,9 +211,17 @@ export default function PortfolioScreen({ navigation }: Props) {
...
+        stickyHeaderIndices={[2]}
+        onScroll={Animated.event(
+          [
+            {
+              nativeEvent: {
+                contentOffset: { y: scrollY },
+              },
+            },
+          ],
+          { useNativeDriver: true },
```
[the remark this answers is not in the document]
Comment (LFBarreto) [reply src/screens/Portfolio/index.js:223]: just in case :laughing:

**Review by machard — DISMISSED** [review]

**Review by JunichiSugiura — DISMISSED** [review]

**Review by Arnaud97234 — DISMISSED** [review]
We can merge.
I left a comment on https://ledgerhq.atlassian.net/browse/LL-5136 suggesting an improvement.

**Review by Arnaud97234 — COMMENTED** [review]
Swap and Buy available from portfolio.
From an account Buy or Swap redirect with preselecting the corresponding currency.

Ok merge ✅

**Review by Arnaud97234 — APPROVED** [review]

**Review by gre — APPROVED** [review]

**Pull request merged**